### PR TITLE
Add users column to enrollment unit and module assignment - this help…

### PR DESCRIPTION
**Pull request by mehalshah**
…s us to faster lookups

### CODE REVIEW

**Comment by mehalshah** [comment]
Looking at travis failure

**Comment by mehalshah** [comment]
Travis failure is dealt with - this is fine to review

File: dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class AddUsersToUnitAndModuleAssignments < ActiveRecord::Migration
+  def change
+    add_reference :plc_enrollment_unit_assignments, :user, index: true
+    add_reference :plc_enrollment_module_assignments, :user, index: true
+
+    Plc::EnrollmentUnitAssignment.all.each do |unit_assignment|
+      unit_assignment.update(user: unit_assignment.plc_user_course_enrollment.user)
```
Comment (joshlory) [inline dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb:7]: Will these updates work when migrating down?

Comment (mehalshah) [inline dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb:7]: All of these are rollback-able I believe, but I'll try it out once I'm out of the DSL branch. If not, I'll make this an up with a down migration to drop the above two columns.

File: dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class AddUsersToUnitAndModuleAssignments < ActiveRecord::Migration
+  def change
+    add_reference :plc_enrollment_unit_assignments, :user, index: true
+    add_reference :plc_enrollment_module_assignments, :user, index: true
```
Comment (joshlory) [inline dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb:4]: Should these be foreign keys?

**Comment by joshlory** [comment]
A few concerns about the migration.  Otherwise LGTM.

File: dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb (hunk 2 above)
Comment (mehalshah) [inline dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb:4]: I've been avoiding them for enrollment stuff, and really most plc stuff. Trying to remember why

File: dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb (hunk 1 above)
Comment (mehalshah) [inline dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb:7]: Confirmed that this rolls back just fine

**Pull request closed without merging**